okpt/io/config/parsers/opensearch.py

```diff
@@ -19,13 +19,23 @@
 Classes:
     OpenSearchParser: OpenSearch config parser.
 """
+from dataclasses import dataclass
 from io import TextIOWrapper
 from typing import Any, Dict
 
 from okpt.io.config.parsers import base
 from okpt.io.utils import reader
 
 
+@dataclass
+class OpenSearchConfig:
+    index_spec: Dict[Any, Any]
```

Comment:
Why is this Any, Any as opposed to String, Any?

Reply:
Updated to `[str, Any]`